packages/next/src/bundles/webpack/packages/HotModuleReplacement.runtime.js

```diff
@@ -3,407 +3,407 @@
 	Author Tobias Koppers @sokra
```

Comment:
Not sure how these files got formatted, they are supposed to be ignored by Prettier <picture data-single-emoji=":surprised-wide-eyed:" title=":surprised-wide-eyed:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..RydF9gwGcFucFhtB.sswWbvMntLZfeIJI7cGlUwJifZG45fsTtDIqbai5nz4DGdUdvEt6fcugHUXY7ZjAWc7EqTFkKq2Vw9CcTmk2KUQ2iER3zwDdMyVId2U99Wd2ZlkWE9f4Kj7_hdSTSMEcWR_lJTPv.Q7SVrprhPe3MpkXxR0Czcg" alt=":surprised-wide-eyed:" align="absmiddle"></picture>:

https://github.com/vercel/next.js/blob/12bfabd179838918560e058ddf0b5213e87b66c1/.prettierignore#L5-L6

I don't have these changes locally either, so something is off with my commit hook, maybe?